### PR TITLE
Lead maintainer

### PR DESCRIPTION
**Pull request by diasdavid**
For https://github.com/ipfs/community/pull/301

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "multiaddr",
   "version": "5.0.0",
   "description": "multiaddr implementation (binary + string representation of network addresses)",
+  "lead-maintainer": "Victor Bjelkholm <victorbjelkholm@gmail.com>",
```
Comment (VictorBjelkholm) [inline package.json]: shouldn't this be `leadMaintainer` to follow the rest of the casing of keys in package.json?

Comment (diasdavid) [reply package.json]: Good point. Now I've to change it all xD

File: package.json [hunk 2]
```diff
@@ -17,10 +17,6 @@
     "coverage-publish": "aegir coverage publish",
     "docs": "aegir docs"
   },
-  "pre-commit": [
```
Comment (vmx) [inline package.json]: This should not be removed, but changed to `pre-push`. I'd also like to see a better commit message :)

Comment (diasdavid) [reply package.json]: @VictorBjelkholm can you handle the changes?

**Pull request merged**